cds_ils/importer/documents/importer.py

```diff
@@ -208,9 +208,28 @@ def update_document(self, matched_document):
             click.secho(e.original_exception.message, fg="red")
             db.session.rollback()
 
+    def validate_found_matches(self, not_validated_matches):
+        """Validate matched & parsed documents have same title/isbn pair."""
+        matches = []
+        partial_matches = []
+        document_class = current_app_ils.document_record_cls
+        title = self.json_data.get("title")
+
+        for pid in not_validated_matches:
+            document = document_class.get_record_by_pid(pid)
+            document_title = document['title']
+
+            if document_title == title:
```

Comment:
there will be a problem with case sensitivity here, for this reason lowercase both titles for the comparison